gulpfile.js

```diff
@@ -156,7 +156,7 @@ if (!argv.production) {
 
     gulp.task('clean', ['clean:build', 'clean:dist', 'clean:report']);
 
-    gulp.task('test:typescript', function(done) {
+    gulp.task('test:typescript', ['build'], function(done) {
```

Comment:
Nein:
`START:
30 07 2016 14:43:21.670:WARN [web-server]: 404: /base/strassengezwitscher/static/build/@angular/core/testing.js
30 07 2016 14:43:21.674:WARN [web-server]: 404: /base/strassengezwitscher/static/build/@angular/platform-browser-dynamic/testing.js
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  Error: (SystemJS) XHR error (404 Not Found) loading http://localhost:9876/static/@angular/core/testing.js
  	Error loading http://localhost:9876/static/@angular/core/testing.js as "@angular/core/testing" from http://localhost:9876/static/frontend/test-helpers/setup.js`

Deshalb ist mindestens noch `copy:staticfiles` nötig. Würde dann bei `build` bleiben